### PR TITLE
Avoid line breaks in descriptions and validate doc_url

### PR DESCRIPTION
**Pull request by delvedor**
As titled.

Related: https://github.com/elastic/elasticsearch-specification/pull/550

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
👍  LGTM. Thanks!

**Review by sethmlarson — APPROVED** [review]
LGTM!

**Review by lockewritesdocs — APPROVED** [review]
LGTM 👍 Thanks @delvedor!

File: output/schema/schema.json [hunk 1]
```diff
@@ -111810,7 +111810,7 @@
       ],
       "query": [
         {
-          "description": "Specifies what to do when the request:\n- Contains wildcard expressions and there are no models that match.\n- Contains the _all string or no identifiers and there are no matches.\n- Contains wildcard expressions and there are only partial matches.",
+          "description": "Specifies what to do when the request: - Contains wildcard expressions and there are no models that match. - Contains the _all string or no identifiers and there are no matches. - Contains wildcard expressions and there are only partial matches.",
```
Comment (swallez) [inline output/schema/schema.json:111813]: Why don't we allow line breaks in descriptions? The above was a list and will be output as a weird single line with dashes.

We should settle on a markup language for descriptions that allows formatting for long descriptions. Considering the use of lists and `` `code` `` snippets, I would suggest CommonMark that has implementations in most languages and can thus be used in code generators.

**Pull request merged**